<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widget</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>146</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_ipserveur">
        <property name="text">
         <string>IP Serveur</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ipserveur">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>172.18.58.135</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_port">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_portserveur"/>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_text">
        <property name="text">
         <string>Text Ã  envoyer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_textenvoyer"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="pushButton_connection">
       <property name="text">
        <string>Connection</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_envoyer">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Envoyer</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_quitter">
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_message">
      <property name="text">
       <string>Message</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_message"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_quitter</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
